transcription disc scripts/moverestorations.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# microservice to move older transcription disc restorations into the restoration_old directory after a newer restoration is made using the mm/makederiv microservice
+
+# color codes for messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+_usage(){
+    echo -e ${GREEN}"\nThis script moves older transcription disc restorations into the restoration_old directory after a newer restoration is made using the mm/makederiv microservice.\n"${NC}
+}
+
+_usage
+
+# asks for a directory and checks it
+
+echo "Enter the directory path to begin:"
+read -r -p "> " user_directory
```

Comment:
Can you adjust so that the script would run without any interaction. For instance, running `./script dir` rather than `./script` and being prompted for a dir. Then part 2, make it loop over input directories, as in `./script dir1 dir2 ....`